crowbar_framework/app/models/ceilometer_service.rb

```diff
@@ -119,6 +129,10 @@ def validate_proposal_after_save proposal
 
     validate_minimum_three_nodes_in_cluster(proposal)
 
+    unless proposal["deployment"]["ceilometer"]["elements"]["ceilometer-hyperv-agent"].empty? || hyperv_available?
```

Comment:
I think I got it. It has to be `ceilometer-agent-hyperv` and not `ceilometer-hyperv-agent`